plone/rfc822/fields.rst

```diff
@@ -86,7 +80,7 @@ This interface is implemented by a the following class:
     ...     _id = 'some.id'
     ...     _dottedName = 'dotted.name'
     ...     _bool = True
-    ...     _int = long(-10)
+    ...     _int = long(-10) if six.PY2 else -10
```

Comment:
Prefer to check for PY2: /cc @pbauer 

https://eev.ee/blog/2016/07/31/python-faq-how-do-i-port-to-python-3/#making-some-tough-decisions